test/test_jit.py

```diff
@@ -11877,18 +11874,23 @@ def test_lstm_cuda(self):
         self.assertGraphContainsExactly(
             forward_graph, 'prim::FusionGroup', 1, consider_subgraphs=True)
         self.assertExpectedGraph(forward_graph, subname='forward')
+        self.assertTrue(len(forward_graph.nodes()) == 2)
+        FileCheck().check("DifferentiableGraph").check_next("TupleConstruct") \
+            .check_next("return").run(str(forward_graph))
 
         hy, cy = module(*inputs)
         (hy + cy).sum().backward()
-        self.assertExpectedGraph(backward_graph(module), subname='backward')
+        backward = backward_graph(module)
+        FileCheck().check("FusionGroup_0").check_next("FusionGroup_1").run(str(backward))
```

Comment:
The automated checking is really nice for asserting properties of the forward/backward graphs. In addition to those checks, could we still keep around the forward / backward expect files for this lstm? They're pretty useful as documentation and help debugging and figuring out exactly what is going on with the graphs.

Reply:
We don't want to test against the expect files anymore. Maybe we could do something like self.updateExpectFile(str(graph)) that doesn't fail if there's a difference ?

Reply:
Yes, it would be fine to not fail on difference. `self.updateExpectFile(str(graph))` is still going to race, though :(

Reply:
It will still race but I'm not sure of how to keep around expect files without racing... at least this won't fail master